### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by tzekid** [zzuussee/TorDroid#3]
From an outsider perspective,
it would be nice to have a rough documentation on how everything is tied up together
and a (more) concise TODO list in the Projects section.

**Comment by zzuussee** [comment]
Yes you're right. The project related TODO list is actually just meant as a rough sketch. But a appropriate README.md with proper explanations of use and workflow would definitely help the cause. If you would like to participate more deeply in our project then write me an official mail. We would like to have you officially on board of this project.
